src/main.py

```
@@ -193,39 +198,64 @@ async def handle_message(update: Update, context: ContextTypes.DEFAULT_TYPE):  #
+            if THROTTLE:
+                time.sleep(5)
+
+            # Send the picture to the chat
+            await send_pic(update, pic)
 
     finally:
-        # Clean up temporary files
-        if video_path and os.path.exists(video_path):
-            cleanup(video_path)
+        if media_path:
+            cleanup_file(media_path)
```

Ruff (0.8.2)

Undefined name `cleanup_file`

(F821)